Sources/LeafKit/LeafToken.swift

```diff
@@ -1,5 +1,22 @@
 public enum Keyword: String, Equatable {
     case `in`, `true`, `false`, `self`, `nil`, `yes`, `no`
+    
+    var isBooleanValued: Bool {
+        switch self {
+            case .true,
+                 .false
+                 : return true
+            default: return false
+        }
+    }
+    
+    var booleanValue: Bool? {
+        switch self {
+            case .true: return true
+            case .false: return false
+            default: return nil
+        }
+    }
```

Comment:
### Question ###
What are yes and no supposed to be used for? Should they be considered boolean-comparable?

Reply:
I think the idea here is you could do:

```leaf
#if("yes"):

#endif
```

I think Leaf 3 supported this so we should probably leave it.

Reply:
Right - but would we consider it to be something that's actually *returnable* as a value-type?

EG - should #(true) evaluate to .raw(true) (I say yes), and should #(yes) evaluate to .raw(true) (maybe?) whereas #("yes") would definitely evaluate to .raw(yes)

Reply:
`#if(yes)` not `#if("yes")` since #2 would be a stringLiteral and not a keyword

Reply:
Have updated to make `yes` and `no` keywords satisfy `isBooleanValued` and return true/false